bullet_train/app/views/devise/registrations/_two_factor.html.erb

```diff
@@ -1,62 +1,64 @@
-<%= render 'account/shared/box', divider: @backup_codes do |box| %>
-  <% box.title t("users.edit.two_factor.header") %>
-  <% box.description t("users.edit.two_factor.description_#{@user.otp_required_for_login? ? 'enabled' : 'disabled'}") %>
-  <% if current_user.otp_secret %>
-    <% if @backup_codes %>
-      <% box.body do %>
-        <%= render 'account/shared/alert' do %>
-          <%= t('users.edit.two_factor.warning').html_safe %>
-        <% end %>
+<%= turbo_frame_tag "two-factor-frame" do %>
```

Comment:
This diff for this file looks messier than it is because I wrapped the whole file in a `turbo_frame_tag` so everything got indented by a level. Turning on "hide whitespace" here will make it easier to see what's going on:

![CleanShot 2025-05-30 at 13 21 58](https://github.com/user-attachments/assets/735f3db3-d081-4b5e-b67d-b0e8eb6b29da)